templates/apex/index.json

```diff
@@ -280,7 +280,7 @@
             "text_position_mobile": "center",
             "button_url_1": "booqable://products/",
             "button_url_2": "",
-            "title": "<p>Join us for<br>a track day</p>",
+            "title": "Join us for<br> a track day",
```

Comment:
I don't think we should not use `br` tags in data, because the `text` field type doesn't allow multiline text and customers will not be able to replicate the same from the theme editor. Could you check other files for the same?

Reply:
After Chameleon theme QA, the Product team decided to convert `text` field into `contentEditor`. It was made for flexibility and the customers' convenience, they can break lines more easily and prevent the elongation of titles. 
In this PR I just restored the code to its initial format with a text field and `<br>` tags so that the text looks closer to the design.
On the other hand, if we remove `<br>` tags we lose a chance to demonstrate to our customers the option of text positioning in the section especially with long titles.
I think we can leave it as it is now because it's only demo, and the customers will change the text definitely

Reply:
I'm afraid that these breaks may cause confusion on further editing by customers who will ask in the future if it's possible to break the text while the input type is single-line text

Reply:
Yes, you're right but there is a small tip for customers below that helps to prevent any confuses I believe

![Teampaper_2024-03-26 15-09-52@2x](https://github.com/booqable/tough-theme/assets/40244261/523cea32-9c59-4c00-9ef4-90fe127668a3)

Reply:
> Yes, you're right but there is a small tip for customers below that helps to prevent any confuses I believe
> 
> ![Teampaper_2024-03-26 15-09-52@2x](https://private-user-images.githubusercontent.com/40244261/316874913-523cea32-9c59-4c00-9ef4-90fe127668a3.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE1MjY5NjgsIm5iZiI6MTcxMTUyNjY2OCwicGF0aCI6Ii80MDI0NDI2MS8zMTY4NzQ5MTMtNTIzY2VhMzItOWM1OS00YzAwLTllZjQtOTBmZTEyNzY2OGEzLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI3VDA4MDQyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVlZWM0YjhmOGYwMTk0MjllZTA2Y2Q1YzY5MzEzNGM1OGZiMzk0MTcyN2I0ZjFkMWExZGEzN2I4MzE2ZmVjYWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.29jhwpUI1tQHf7-HtBEV535xAJ0isbyighGo6OkxDy0)

Should we use `<br />` here? As far as I remember all tags should be closed

Reply:
I just checked HTML standards [here](https://www.w3schools.com/tags/tag_br.asp) and the `<br>` tag is used without space and slash